tests/modules/test_morpheus_modules.py

```diff
@@ -345,10 +344,99 @@ def _on_next(control_msg):
     for f in files:
         os.remove(f[0])
 
+def test_rest_loader_module():
```

Comment:
Lets expand this out a bit; you're testing a few different things in this one unit test. I'd like to see a few different scenarios spread across multiple tests.